src/cloud-api-adaptor/ibmcloud/ROKS_SETUP.md

```diff
@@ -204,7 +204,36 @@ sed -i ".bak" -e 's/DISABLECVM="true"/DISABLECVM="false"/' -e 's/bx2-2x8/bx3dc-2
 ```
 
 > [!WARNING]
-> Confidential (CVM) mode is still a work in progress and should only be enabled for development purposes at this time.
+> If you want to experiment with attestation, you will also need access to a [Trustee](https://github.com/confidential-containers/trustee) service that is configured to verify TDX evidence. You also need to make sure that your configured peer pod VM image includes the TDX runtime libraries and attestation agent.
```

Comment:
Quick question: What do you mean by "TDX runtime libraries"? Presumable the attestation-agent here is referring to the TDX specific TEE_PLATFORM version?

Reply:
@stevenhorsman I changed it to: `the TDX attestation agent and kernel modules`.

Reply:
I think it should either be built into the makefile or documented somewhere with more image building details, but I'm waiting until we understand it a little better after we have it also working in RHEL. I'm imagining a future PR that explains this better and also how to run an example trustee. Just wanted to get something in there now, so that people can at least understand what's needed if they want to also experiment with attestation.

Reply:
That sounds fair enough. Thanks